frontend/src/metabase/dashboard/components/AutoApplyFilterToast/AutoApplyFilterToast.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect, useMemo } from "react";
+import { t } from "ttag";
+
+import { useSelector, useDispatch } from "metabase/lib/redux";
+import {
+  getAutoApplyFiltersToastStateName,
+  getDashboardId,
+  getIsAutoApplyFilters,
+  getParameterValues,
+} from "metabase/dashboard/selectors";
+import {
+  dismissAutoApplyFiltersToast,
+  saveDashboardAndCards,
+  setDashboardAttributes,
+  setNeverShowAutoApplyFiltersToast,
+  setReadyForAutoApplyFiltersToast,
+} from "metabase/dashboard/actions";
+import { addUndo } from "metabase/redux/undo";
+import { ToasterButton } from "metabase/components/Toaster/Toaster.styled";
+
+export default function AutoApplyFilterToast() {
+  const isAutoApplyFilters = useSelector(getIsAutoApplyFilters);
+  const parameterValues = useSelector(getParameterValues);
+  const autoApplyFiltersToastStateName = useSelector(
+    getAutoApplyFiltersToastStateName,
+  );
+  const dashboardId = useSelector(getDashboardId);
+
+  const hasParameterValues = useMemo(
+    () =>
+      Object.values(parameterValues).some(parameterValue =>
+        Array.isArray(parameterValue)
+          ? parameterValue.length > 0
+          : parameterValue != null,
+      ),
+    [parameterValues],
+  );
```

Comment:
Seems in some cases e.g. when we clear the parameter values, the `parameterValues` object isn't empty but has parameter ids with `undefined` or `[]` as values
e.g.
```ts
{
  param_id: null
}
```